packages/orbit-design-tokens/src/index.d.ts

```diff
@@ -0,0 +1,623 @@
+  paletteCloudLight: string;
+  paletteCloudLightHover: string;
+  paletteCloudLightActive: string;
+  paletteCloudNormal: string;
+  paletteCloudNormalHover: string;
+  paletteCloudNormalActive: string;
+  paletteCloudDark: string;
+  paletteInkLighter: string;
+  paletteInkLighterHover: string;
+  paletteInkLighterActive: string;
+  paletteInkLight: string;
+  paletteInkLightHover: string;
+  paletteInkLightActive: string;
+  paletteInkNormal: string;
+  paletteInkNormalHover: string;
+  paletteInkNormalActive: string;
+  paletteOrangeLight: string;
+  paletteOrangeLightHover: string;
+  paletteOrangeLightActive: string;
+  paletteOrangeNormal: string;
+  paletteOrangeNormalHover: string;
+  paletteOrangeNormalActive: string;
+  paletteOrangeDark: string;
+  paletteOrangeDarkHover: string;
+  paletteOrangeDarkActive: string;
+  paletteOrangeDarker: string;
+  paletteRedLight: string;
+  paletteRedLightHover: string;
+  paletteRedLightActive: string;
+  paletteRedNormal: string;
+  paletteRedNormalHover: string;
+  paletteRedNormalActive: string;
+  paletteRedDark: string;
+  paletteRedDarkHover: string;
+  paletteRedDarkActive: string;
+  paletteRedDarker: string;
+  paletteGreenLight: string;
+  paletteGreenLightHover: string;
+  paletteGreenLightActive: string;
+  paletteGreenNormal: string;
+  paletteGreenNormalHover: string;
+  paletteGreenNormalActive: string;
+  paletteGreenDark: string;
+  paletteGreenDarkHover: string;
+  paletteGreenDarkActive: string;
+  paletteGreenDarker: string;
+  paletteBlueLight: string;
+  paletteBlueLightHover: string;
+  paletteBlueLightActive: string;
+  paletteBlueNormal: string;
+  paletteBlueNormalHover: string;
+  paletteBlueNormalActive: string;
+  paletteBlueDark: string;
+  paletteBlueDarkHover: string;
+  paletteBlueDarkActive: string;
+  paletteBlueDarker: string;
+  paletteSocialFacebook: string;
+  paletteSocialFacebookHover: string;
+  paletteSocialFacebookActive: string;
+};
+
+export type ThemePaletteColors = {
+  productLight: string;
+  productLightHover: string;
+  productLightActive: string;
+  productNormal: string;
+  productNormalHover: string;
+  productNormalActive: string;
+  productDark: string;
+  productDarkHover?: string;
+  productDarkActive?: string;
+  productDarker?: string;
+};
+
+type CustomPalette = {
+  product?: Partial<ProductColor>;
+  white?: Partial<WhiteColor>;
+  cloud?: Partial<CloudColor>;
+  ink?: Partial<InkColor>;
+  orange?: Partial<StatusColor>;
+  red?: Partial<StatusColor>;
+  green?: Partial<StatusColor>;
+  blue?: Partial<StatusColor>;
+  social?: Partial<SocialColor>;
+};
+type CustomBase = Partial<Base>;
+
+type CustomFoundation = Partial<{
+  palette: CustomPalette;
+  base: CustomBase;
+}>;
+
+export declare const ConvertHexToRgba: (color: string, opacity: number) => string;
+export declare const FromPlainObject: (themePaletteColors: ThemePaletteColors) => Tokens;
+export declare const GetTokens: (customFoundation?: CustomFoundation) => Tokens;
```

Comment:
Just a quick fix of exported types, did not wanted to do that in the orbit-design-tokens repository
```suggestion
export declare const fromPlainObject: (themePaletteColors: ThemePaletteColors) => Tokens;
export declare const getTokens: (customFoundation?: CustomFoundation) => Tokens;
```